torch/_tensor_str.py

```diff
@@ -266,6 +266,15 @@ def _str(self):
         if values.numel() == 0:
             values_str += ', size=' + str(tuple(values.shape))
         tensor_str = indices_prefix + indices_str + '),\n' + ' ' * indent + values_prefix + values_str + ')'
+    elif self.is_quantized:
+        suffixes.append('size=' + str(tuple(self.shape)))
+        if not has_default_dtype:
+            suffixes.append('dtype=' + str(self.dtype))
+        # TODO: change to a call to self.q_scheme() when we add q_scheme method
+        suffixes.append('quantization_scheme=' + 'per_tensor_affine')
```

Comment:
can we just do a minimal q_scheme thing in this PR?  The issue with this is there is no enforcement mechanism to make us realize something is wrong once we support different qschemes.

Or heck, just make it not print the quantization scheme for now.

Reply:
Sure, let me comment this for now